docs/modpacks/play.md

```diff
@@ -63,3 +63,21 @@ Click the "Add a new Minecraft Core or Mod Pack" tile in the Start Menu, then cl
 ![Add a new Minecraft Core or Mod Pack Tile](../../static/img/tutorial/BakaXLAddCoreTile.png)
 
 Click "Continue" and drag the modpack file into BakaXL.
+
+### Superpack
+
+Superpack is an application for extracting/downloading modpacks to directories.
+
+#### Install through Superpack
+
+Modpacks can be installed by simply navigating or searching the required modpack.
```

Comment:
Confusingly worded. Did you mean "requested" instead of "required"? And you can remove the "navigating" part, since searching *is* navigating.